src/istioctl/istioctl.go

```diff
@@ -203,17 +203,24 @@ func ExecWithWriters(homeDir string, args []string, stdout, stderr io.Writer) er
 	return cmd.Run()
 }
 
-func Fetch(homeDir string, version, flavor string, flavorVersion int, ms *api.Manifest) (*api.IstioDistribution, error) {
-	d, err := processFetchParams(version, flavor, flavorVersion, ms)
+func Fetch(homeDir string, name, version, flavor string, flavorVersion int, ms *api.Manifest) (*api.IstioDistribution, error) {
+	d, err := processFetchParams(name, version, flavor, flavorVersion, ms)
 	if err != nil {
 		return nil, err
 	}
 
 	return d, fetch(homeDir, d, ms)
 }
 
-func processFetchParams(version, flavor string, flavorVersion int,
+func processFetchParams(name, version, flavor string, flavorVersion int,
 	ms *api.Manifest) (*api.IstioDistribution, error) {
+	if len(name) != 0 {
+		d, err := api.IstioDistributionFromString(name)
+		if err != nil {
+			return nil, fmt.Errorf("cannot parse given name to %s istio distribution", name)
```

Comment:
```suggestion
			return nil, fmt.Errorf("cannot parse given name %s  to istio distribution", name)
```

Reply:
Could you do the same fix for https://github.com/tetratelabs/getistio/blob/ddc6cb8c3305eae74732fa7500fe5e64ed9bc59d/cmd/switch.go#L91 ?